services/static-webserver/client/Makefile

```diff
@@ -52,6 +52,7 @@ compile: ## qx compiles host' 'source' -> image's 'build-output'
 		--build-arg VCS_STATUS_CLIENT=${VCS_STATUS_CLIENT} \
 		--build-arg VCS_URL=${VCS_URL} \
 		--target=build-client .
+	python ./scripts/post-compile.py
```

Comment:
how did you solve the dev part at the end? I do not see the changes here

Reply:
True the metadata thing can stay as is, but the routes need to be updated.